### PR TITLE
Update Accept header to application/vc+ld+json

### PR DESCRIPTION
**Pull request by rhofvendahl**
This PR updates references to `Accept` header in "Create Credentials" requests to use either `application/vc+ld+json` or `application/vc+ld+json+sd-jwt`, as discussed in issue https://github.com/w3c-ccg/traceability-interop/issues/574.

EDIT: using `application/sd-jwt` rather than `application/vc+ld+json+sd-jwt`, as discussed in 11/7/23 trace call.

### CODE REVIEW

**Review by brownoxford — APPROVED** [review]

**Comment by TallTed** [comment]
As we discussed on the call --

`application/vc+ld+json+sd-jwt` should not exist.

An `application/sd-jwt` is akin to an `application/zip`, and should not externally expose the media type of the document(s) it contains in its own media type; those should be revealed/discovered when that/those document(s) are extracted or sniffed.

In this scenario, an `application/vc+ld+json` document would be the result of extracting/sniffing from an `application/sd-jwt` document, and would be what is expected during such a transaction. Contents of other media types should lead to errors.

In sum: `Accept` headers in "Create Credentials" requests should be either `application/vc+ld+json` or `application/sd-jwt` (the latter of which will always package an `application/vc+ld+json`, in this context).

**Review by mkhraisha — APPROVED** [review]

**Comment by rhofvendahl** [comment]
I'll be splitting out the data integrity proof updates, with sd-jwt work pending resolution in the jose-cose working group

File: docs/tutorials/vc-jwt/vc-jwt.collection.json [hunk 1]
```diff
@@ -156,7 +156,7 @@
 				"header": [
 					{
 						"key": "Accept",
-						"value": "application/json",
```
Comment (OR13) [inline docs/tutorials/vc-jwt/vc-jwt.collection.json:159]: I would remove this part

Comment (rhofvendahl) [reply docs/tutorials/vc-jwt/vc-jwt.collection.json:159]: Could you say more? In this case the `Accept` header is on a `Verify` request, so just represents a preference that the verification response be in json format, right?

**Comment by rhofvendahl** [comment]
@mkhraisha Once Orie's suggestion is resolved this should be ready to merge.

There are still some references to `application/sd-jwt` which may need to change if the jose-cose working group decides to go with something else, but there needs to be _some_ value in those cases so I think it should work for now.

**Review by OR13 — APPROVED** [review]

**Comment by OR13** [comment]
Related PR: https://github.com/w3c/vc-jose-cose/pull/186

^ in case this PR is not merged, the current PR will continue to be correct.

**Pull request merged**